packages/core/src/entities/feature.ts

```diff
@@ -75,30 +75,79 @@ export class RuntimeFeature<T extends Feature, ENV extends AnyEnvironment> {
     }
 }
 
+/**
+ * Feature is a class that can be used to create a runtime feature. It is used to define the feature and its dependencies.
+ *
+ * @class Feature
+ * @template ID - Way to identify the feature. Provide a unique string or symbol.
+ * @template Deps - List of dependencies.
+ * @template API - API that the feature exposes.
+ * @template EnvironmentContext - Context that the feature is running in.
+ */
 export class Feature<
     ID extends string = string,
     Deps extends Feature[] = any[],
     API extends EntityRecord = any,
     EnvironmentContext extends Record<string, DisposableContext<any>> = any
 > {
+    /**
+     * Identifier of the feature.
+     *
+     * @type {Feature<ID, Feature[], API, EnvironmentContext>}
```

Comment:
lets avoid this repetition. we already write the type in the code.
we should probably use tsdoc: https://tsdoc.org/